lib/library.cpp

```diff
@@ -663,12 +663,16 @@ Library::Error Library::loadFunction(const tinyxml2::XMLElement * const node, co
             }
             for (const tinyxml2::XMLElement *argnode = functionnode->FirstChildElement(); argnode; argnode = argnode->NextSiblingElement()) {
                 const std::string argnodename = argnode->Name();
+                int indirect = 0;
+                const char * const indirectStr = node->Attribute("size");
```

Comment:
Should the string literal say `"indirect"` ?

Reply:
What name would you suggest?

Reply:
I think your suggestion "indirect" is fine. So feel free to change `node->Attribute("size")` to `node->Attribute("indirect")`.